packages/adminrouter/extra/src/docs/api/nginx.master.yaml

```diff
@@ -0,0 +1,332 @@
+ngindox: 0.1.0
+backends:
+  auth:
+    server: '127.0.0.1:8101'
+    name: DC/OS Authentication (OAuth)
+    reference: 'https://dcos.io/docs/1.9/administration/id-and-access-mgt/iam-api/'
+  cosmos:
+    server: '127.0.0.1:7070'
+    name: DC/OS Package Manager (Cosmos)
+    reference: 'https://dcos.io/docs/1.9/usage/managing-services/package-api/'
+  dcos_history_service:
+    server: 'leader.mesos:15055'
+    name: DC/OS History
+    reference: 'https://github.com/dcos/dcos/tree/master/packages/dcos-history/extra#api'
+  dddt:
+    server: '127.0.0.1:1050'
+    name: DC/OS Diagnostics (3DT)
+    reference: >-
+      https://dcos.io/docs/1.9/administration/monitoring/#system-health-http-api-endpoint
+  exhibitor:
+    server: '127.0.0.1:8181'
+    name: Exhibitor (Zookeeper)
+    reference: 'https://github.com/soabase/exhibitor/wiki/REST-Introduction'
+  log:
+    server: 'unix:/run/dcos/dcos-log.sock'
+    name: DC/OS Log
+    reference: 'https://dcos.io/docs/1.9/administration/logging/logging-api/'
+  marathon:
+    server: 'master.mesos:8080'
+    name: Marathon
+    reference: 'https://dcos.io/docs/1.9/usage/managing-services/rest-api/'
+  mesos:
+    server: 'leader.mesos:5050'
+    name: Apache Mesos
+    reference: 'https://mesos.apache.org/documentation/latest/endpoints/'
+  mesos_dns:
+    server: 'master.mesos:8123'
+    name: Mesos DNS
+    reference: 'https://dcos.io/docs/1.9/usage/service-discovery/mesos-dns/http-interface/'
+  metrics:
+    server: 'unix:/run/dcos/dcos-metrics-master.sock'
+    name: DC/OS Metrics
+    reference: 'https://dcos.io/docs/1.9/administration/monitoring/metrics/metrics-api/'
+  navstar:
+    server: '127.0.0.1:62080'
+    name: Navstar
+  pkgpanda:
+    server: 'unix:/run/dcos/pkgpanda-api.sock'
+    name: DC/OS Component Package Manager (Pkgpanda)
+    reference: 'https://dcos.io/docs/1.9/administration/component-management/'
+routes:
+  /:
+    group: Root
+    matcher: path
+    description: DC/OS GUI
+    file: /opt/mesosphere/active/dcos-ui/usr
+    path: /
+  '/(slave|agent)/(?<agentid>[0-9a-zA-Z-]+)':
+    group: Agent
+    matcher: regex
+    description: Redirect to add trailing slash
+    visibility: hidden
+    rewrites:
+      - regex: ^/(slave|agent)/(.*)$
+        replacement: /agent/$2/
+        type: last
+    path: '/(slave|agent)/(?<agentid>[0-9a-zA-Z-]+)'
+  '/(slave|agent)/(?<agentid>[0-9a-zA-Z-]+)(?<url>.+)':
+    group: Agent
+    matcher: regex
+    description: API proxy to a specific agent node
+    proxy:
+      path: '$agentaddr:$agentport'
+    rewrites:
+      - regex: '^/(slave|agent)/[0-9a-zA-Z-]+/.*$'
+        replacement: $url
+        type: break
+    path: '/(slave|agent)/(?<agentid>[0-9a-zA-Z-]+)(?<url>.+)'
+  /acs/api/v1:
+    group: Authentication
+    matcher: path
+    description: Access Control Service
```

Comment:
'Access Control Service' is a legacy term from Enterprise DC/OS. I think here it might be more correct to say "DC/OS Authentication (OAuth)" (same as above).